src/backend/base/langflow/base/agents/agent.py

```diff
@@ -44,18 +44,21 @@ class LCAgentComponent(Component):
             display_name="Handle Parse Errors",
             value=True,
             advanced=True,
+            info="Should the Agent fix errors when reading user input for better processing?",
         ),
         BoolInput(
             name="verbose",
             display_name="Verbose",
             value=True,
             advanced=True,
+            info="Print more details about the Agent's actions and thoughts.",
```

Comment:
I'm not sure about this one. 
This is mostly to print stuff in the terminal. Should we remove this?